proposals/0339-module-aliasing-for-disambiguation.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SE-0339](0339-module-aliasing-for-disambiguation.md)
 * Authors: [Ellie Shin](https://github.com/elsh)
 * Review Manager: [John McCall](https://github.com/rjmccall)
-* Status: **Accepted**
+* Status: **Implemented (Swift 5.7)** 
 * Pitch: [Module Aliasing](https://forums.swift.org/t/pitch-module-aliasing/51737)
 * Implementation: ([toolchain](https://github.com/apple/swift/pull/40899)),
 [apple/swift-package-manager#4023](https://github.com/apple/swift-package-manager/pull/4023), others
```

Comment:
We'd better keep this line in the same format as other proposals🤔

Reply:
would you like to open a PR to fix it?

Reply:
I'm going to merge this PR regardless; I'm not sure we're strict about the Implementation line.